content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -262,6 +267,12 @@ Enable and start `kubelet`:
 ```bash
 systemctl enable --now kubelet
 ```
+
+{{< note >}}
+Flatcar Linux have `/usr` mounted as read-only and additional steps that needs
```

Comment:
Hello @rikatz . The link should be changed. Here are some additional suggested changes:

```
The Flatcar Linux distribution mounts the `/usr` directory as a read-only filesystem.
Before bootstrapping your cluster, you need to take additional steps to configure a writable directory.
See the [Kubeadm Troubleshooting guide](/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/#usr-mounted-read-only/) to learn how to set up a writable directory.

```

Reply:
Agree, changed! Thanks

Reply:
Thanks @rikatz .
I looked at the preview page. The link in the tab component does not appear to be displaying as link.
The link is correct and you can click on the text, but the text does not appear as a link. I will log an issue.
Otherwise, I think the changes look good.
@neolit123 , Would you have time for a quick review? Thanks!

Reply:
thanks for the updates and review!
/lgtm